.devcontainer/devcontainer.json

```diff
@@ -12,7 +12,7 @@
 		// "BIND_LOCALHOST_DOCKER": "true"
 	},
 	"extensions": [
-		"ms-vscode.go",
+		"golang.go",
```

Comment:
Why this change?

Reply:
The ms-vscode extension doesn't exists anymore. The ms-vscode repo (https://github.com/microsoft/vscode-go) was archived a while ago with a note that https://github.com/golang/vscode-go is now the new home.

Reply:
I think this change should better place in other PR, the choice about recommended extensions need more discuss.

Reply:
@daixiang0 I have rolled back the change and will create a new pull request